internal/test/test.go

```diff
@@ -26,7 +27,8 @@ import (
 func CmdValidator(t *testing.T, subject *cobra.Command, nSubCommands int, flags []string) {
 	t.Helper()
 	a := assert.New(t)
-	a.Len(subject.Commands(), nSubCommands)
+	assertMsg := fmt.Sprintf("Sub command count mismatch. Expected %d, got %d. Check the CmdValidator for your command.", len(subject.Commands()), nSubCommands)
+	a.Len(subject.Commands(), nSubCommands, assertMsg)
```

Comment:
[not-blocking] do you want to change this to plain old go?

```suggestion
	if len(subject.Commands()) != nSubCommands {
		t.Errorf("Sub command count mismatch. Expected %d, got %d. Check the CmdValidator for your command.\n", len(subject.Commands()), nSubCommands)
	}
```

Reply:
Ah cool, I'll do that in the next PR as a flyby